source/server/admin/listeners_handler.cc

```diff
@@ -22,12 +22,15 @@ Http::Code ListenersHandler::handlerDrainListeners(Http::ResponseHeaderMap&,
                                                  : ListenerManager::StopListenersType::All;
 
   const bool graceful = params.find("graceful") != params.end();
+  const bool skip_exit = params.find("skip_exit") != params.end();
```

Comment:
It seems we should add new param here also https://github.com/envoyproxy/envoy/blob/a5930b27b083d71a92ca66ec5200cca65721f0a7/source/server/admin/admin.cc#L200